packages/flutter/lib/src/painting/text_painter.dart

```diff
@@ -583,8 +583,15 @@ class _LineCaretMetrics {
 /// changes, return to step 2. If the text to be painted changes,
 /// return to step 1.
 ///
-/// The default text style is white. To change the color of the text,
-/// pass a [TextStyle] object to the [TextSpan] in `text`.
+/// > [!IMPORTANT]
```

Comment:
I don't think dartdoc supports this markdown extension. Even if it does, this doesn't feel THAT important to warrant a highlighted section.

Reply:
Dartdoc does now support this - can't link it right now, but search for "> [!IMPORTANT]" with quotes and you'll get to the issue/PR. Whether or not it's good to use here, I feel it is - in my case, it wasn't immediately clear why the texts were different colours since I couldn't see the background.

Reply:
oh https://github.com/dart-lang/dartdoc/issues/3995. TIL thank you!

Reply:
Wouldn't "The default text style color is white on non-web platforms and black on the web" suffice? For API docs I would prefer less reading if we can convey the same message.

Reply:
I've reduced it, using that text for the first line. There's still the second part, since I think that's also important, but I've also reduced that.

Reply:
I don't think the repo style guide provides suggestions regarding markdown highlights, but I personally think it doesn't add much value here, and it could be distracting. For most readers the life cycle / how-to information is much more interesting / important. We can add highlights back if more people are actually relying on the default text color and are surprised by the default being different on web/non-web platforms.

Reply:
Removed the highlight. I do think that this could be more visible still somehow. If I was having the issue, I'm not sure `TextPainter` is the first place I'd look.